packages/browser/src/types/provider.ts

```diff
@@ -18,4 +18,5 @@ export interface ExperimentUserProvider {
  */
 export interface ExperimentAnalyticsProvider {
   track(event: ExperimentAnalyticsEvent): void;
+  unset(event: ExperimentAnalyticsEvent): void;
```

Comment:
This is going to break the smart recruiters custom implementation. We should technically release a v2...

Maybe we discuss the change with them and work through their implementation together? I'd like to avoid publishing v2 until we overhaul the library. 😕

Alternatively, we could have the logic built into the event in track like we do with set user properties. and unset the property if the variant value is null (unassigned or fallback).

Reply:
I made this function optional, which should make it backwards compatible